solidity/contracts/isms/native/PolygonISM.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
The bridges keep on coming, awesome! Can we split them up into separate PRs?

Reply:
Split into op, arb and polygon or stop at polygon and make another PR for Gnosis, etc?